holoviews/plotting/bokeh/chart.py

```diff
@@ -35,6 +35,9 @@ class PointPlot(LegendPlot, ColorbarPlot):
         The current selection as a list of integers corresponding
         to the selected items.""")
 
+    hit_dilation = param.Integer(default=1, bounds=(1, None), doc="""
```

Comment:
Should probably just be added to `style_opts` rather than making it a plot parameter.

Reply:
I did not know that it would not be applied in style_opts if it was also a parameter. I have added it as style_opts now.

Reply:
Sorry if that wasn't clear.

Reply:
It was clear. Before your comment, I had tried it as a parameter and also added it as `style_opts`, which did not work. After your comment things clicked.